build.symbian/pjsip_ua.mmp

```diff
@@ -40,6 +40,7 @@ SOURCE	sip_replaces.c
 SOURCE	sip_xfer.c
 SOURCE	sip_100rel.c
 SOURCE	sip_timer.c
+SOURCE	sip_siprec.c
```

Comment:
no need to modify Symbian build. It's deprecated and no longer used.

Reply:
fix it